tests/study_tests/test_multi_objective.py

```diff
@@ -0,0 +1,121 @@
+from typing import Sequence
+
+import pytest
+
+import optuna
+from optuna.study import StudyDirection
+from optuna.study._multi_objective import _dominates
+from optuna.trial import FrozenTrial
+from optuna.trial import TrialState
+
+
+def _create_trial(values: Sequence[float], state: TrialState = TrialState.COMPLETE) -> FrozenTrial:
+    return optuna.trial.create_trial(values=list(values), state=state)
```

Comment:
IMO, we can remove this helper function by `from optuna.trial import create_trial`. This is because the values pass to this functions are always lists, and the default value of `state` is `TrialState.COMPLETE`.

https://optuna.readthedocs.io/en/latest/reference/generated/optuna.trial.create_trial.html#optuna.trial.create_trial

Reply:
Agree. It looks enough to call `create_trial()` directly.